lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1474,7 +1474,7 @@ export class UserAgentApplication {
   /* tslint:disable:no-string-literal */
   private saveAccessToken(response: AuthResponse, authority: string, parameters: any, clientInfo: string, idTokenObj: IdToken): AuthResponse {
       let scope: string;
-      let accessTokenResponse = { ...response };
+      const accessTokenResponse = { ...response };
```

Comment:
On line 1510, we assign new values to accessTokenResponse. We can't make this a const.

Reply:
Const only restricts the memory reference of the top level item so in this case an object is created with a memory reference

When we assign values to the references inside that object we are not changing the memory reference of the accessTokenResponse 
```js
const accessTokenResponse = {};
accessTokenResponse = {} // not allowed
acessTokenResponse.foo = 'bar' //allowed
```